ecosystem/sep-0012.md

```diff
@@ -475,13 +475,9 @@ Anchor has no information on the customer | `404 Not Found`
 
 Allow the wallet to provide a callback URL to the anchor. The provided callback URL will replace (and supercede) any previously-set callback URL for this account.
 
-Whenever the user's `status` field changes, the anchor will issue a POST request to the callback URL. The payload of the POST request will be the same as the response of [`GET /customer`](#customer-get).
+Whenever the user's `status` field changes, the anchor will issue a POST request to the callback URL. 
```

Comment:
Are we removing the requirement to match the `GET /customer` response? The fields defined in the "Callback POST Request" don't match, and thats a breaking change. Can we keep continue to use the `GET /customer` response body as the payload and just add the header for verification?

Reply:
We should add this statement back, it doesn't need to be removed now that we've moved the new fields into the header.

Reply:
The request body defined in the request description doesn't match the `GET /customer` response though, that needs to change too.

Reply:
I described the fields to send in the POST request, they are the same than the `GET /customer` field with the only modification that `account` (and optionally `memo`) is mandatory. This is to make sure that the request cannot be replayed against a different Wallet that may have the same id.

Reply:
This won't protect against replace of a different wallet, because the same account and memo ID might be being used by two wallets.

Reply:
Yes that's why I added the host in the signature as well. I think the host + id is sufficient though. I'll make the modification.

Reply:
@JakeUrban one thing about the parameters of POST request (callback). It still needs to have the `status` field in addition to the fields from `GET /customer` correct? The callback needs to report the KYC status of the user.

Reply:
Yes, it needs to have `status`, but thats included in the `GET /customer` response.

Reply:
Right I was looking at the `/customer` request. Thanks!